compliance/collection/nodescanv2/nodescan.go

```diff
@@ -1,8 +1,11 @@
 package nodescanv2
```

Comment:
A unit test for `nodescan.go` will be very helpful, especially to prove that it does not blow up (e.g. with a runtime panic) when ran in any (potentially unsuitable) context. E.g. on your laptop.